environments/prod/hmcts-access-service-gov-uk.yml

```diff
@@ -3,6 +3,9 @@ ns:
 mx:
 
 A:
+  - name: "@"
+    ttl: 60
+    alias_target_resource_id: "/subscriptions/8cbc6f36-7c56-4963-9d36-739db5d00b27/resourceGroups/lz-prod-rg/providers/Microsoft.Network/frontdoors/hmcts-prod"
```

Comment:
shouldn't this just be a `target_resource_id`?

Reply:
This is just to provide information to the user what this value is for. And this is used as target_resource_id inside the module.